dirty_equals/_datetime.py

```diff
@@ -178,3 +178,99 @@ def __init__(
         )
         if tz is not None:
             self._repr_kwargs['tz'] = tz
+
+
+class IsDate(IsNumeric[date]):
+    """
+    Check if the value is a date, and matches the given conditions.
+    """
+
+    allowed_types = date
+
+    def __init__(
+        self,
+        *,
+        approx: Optional[date] = None,
+        delta: Optional[Union[timedelta, int]] = None,
+        gt: Optional[date] = None,
+        lt: Optional[date] = None,
+        ge: Optional[date] = None,
+        le: Optional[date] = None,
+        unix_number: bool = False,
```

Comment:
I think we should remove this option, it doesn't really make sense to use unix timestamps with dates.